alacritty.yml

```diff
@@ -21,6 +21,14 @@ window:
     columns: 0
     lines: 0
 
+  # Window position (changes require restart)
+  #
+  # Specified in number of pixels.
+  # Comment out to let the window manager handle placement.
```

Comment:
To fall in line with the other documentation comments about commenting-out values, I'd reword this to the following:

```
If the position is not set, the window manager will handle the placement.
```